backend/lib/prometheus/index.js

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright (c) 2018 by SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+'use strict'
+
+const express = require('express')
+const router = module.exports = express.Router()
+const cors = require('cors')
+const { notFound, sendError } = require('../middleware')
+const bodyParser = require('body-parser')
+const client = require('prom-client')
+const collectDefaultMetrics = client.collectDefaultMetrics
+
+// collect metrics every 5 seconds
+collectDefaultMetrics({ timeout: 5000 })
```

Comment:
Don't scrape every 5 seconds - by default we scrape metrics every 30 secs with our Prometheus instances